docs/docs/user-guide/get-started/powershell7-support.md

```diff
@@ -41,10 +41,10 @@ that location or use PowerShell 5.1 to install the modules using 'Install-Module
 Connect-PnPOnline: Could not load file or assembly 'System.IdentityModel.Tokens.Jwt, Version=6.12.2.0, Culture=neutral, PublicKeyToken=31bf3856ad364e35'. Could not find or load a specific file. (0x80131621)
 ```
 
-# PSDesiredStateConfiguration Needs to be Installed Separately
+# PSDesiredStateConfiguration needs to be installed separately
 
-Starting with PowerShell 7.2, the core Desired State Configuration module (PSdesiredStateConfiguration) has been decoupled from the core PowerShell build and now need to be installed separately. In a PowerShell 7+ console, you can install the module by running the command:
+Starting with PowerShell 7.2, the core Desired State Configuration module (PSdesiredStateConfiguration) has been decoupled from the core PowerShell build and now needs to be installed separately. In a PowerShell 7+ console, you can install the module by running the command:
 
 ```powershell
-Install-Module PSDesiredStateConfiguration -Force
+Install-Module PSDesiredStateConfiguration -RequiredVersion 2.0.7 -Force
```

Comment:
I think it's either we add it to the manifest and remove this instruction from here, or leave it out of manifest and keep it here. I am afraid we'll end up having different versions listed in documentation and manifest otherwise.

Reply:
Okay, just a moment. Let me rephrase it so we don't have any ambiguity there.

Reply:
Updated, now using `Update-M365DSCDependencies -Scope AllUsers`, which in itself goes to the manifest and installs the module if it isn't found in the current PowerShell 7 session.

Reply:
@NikCharlebois Can you please have a look at https://github.com/microsoft/DSCParser/pull/50 as well?